cli/cli/out/hook.go

```diff
@@ -0,0 +1,28 @@
+package out
+
+import (
+	"io"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Hook is a hook that writes logs of specified LogLevels to specified Writer
+type Hook struct {
+	Writer    io.Writer
+	LogLevels []log.Level
+	Formatter log.Formatter
+}
+
+func (hook *Hook) Fire(entry *log.Entry) error {
+	line, err := hook.Formatter.Format(entry)
+	if err != nil {
+		return err
+	}
+	_, err = hook.Writer.Write(line)
```

Comment:
I suggest you handle the error here, add the stacktrace and return the propagated error and the in the last line return nil, it's the standard way that we use to handle the errors